scatter.py

```diff
@@ -32,6 +31,9 @@ def make_scatter_plotter(options,data_type1,data_type2,filelist=[]):
     :param data_type2: data type to write on the y-axis.
     :param filelist: list of .json files, if specification of which files to do plotter of is needed, default=[]."""
     filelist = options.get("scatter_files") if options.get("scatter_files") != [] else find_json_files(options) 
+    if options.get("scatter_dir"):
+        for file in range(len(filelist)):
```

Comment:
Is a normal for loop like for file in filelist: possible to use instead?

Reply:
It is and it is Fixed!